esp-hal-smartled/.cargo/config.toml

```diff
@@ -0,0 +1,18 @@
+[target.xtensa-esp32s2-none-elf]
```

Comment:
This shouldn't be required? Not sure why it was added.

Reply:
Without this, the log crate fails to check because atomic instructions are not available for the S2. This (target-specific features here and upstream) is the other reason for the `--target` switches. While there is no code generation, check may emit errors and warnings based on the target.